packages/app/providers/wallet-provider.web.tsx

```diff
@@ -1,17 +1,13 @@
 import { getDefaultWallets, RainbowKitProvider } from "@rainbow-me/rainbowkit";
 import "@rainbow-me/rainbowkit/styles.css";
-import { chain, configureChains, createClient, WagmiConfig } from "wagmi";
+import { configureChains, createClient, allChains, WagmiConfig } from "wagmi";
 import { alchemyProvider } from "wagmi/providers/alchemy";
 import { publicProvider } from "wagmi/providers/public";
 
-const { chains, provider } = configureChains(
-  [chain.polygon],
-  [
-    alchemyProvider({ alchemyId: process.env.NEXT_PUBLIC_ALCHEMY_ID }),
-    publicProvider(),
-  ]
-);
-
+const { chains, provider } = configureChains(allChains, [
```

Comment:
Is it (`allChains`) because the transactions are network agnostic?

Reply:
No, added it to handle a case when wallet doesn't have polygon mumbai chain configured. It was [throwing error] (https://github.com/tmm/wagmi/blob/main/packages/core/src/connectors/injected.ts#L203) on switching.

Reply:
we can also just add polygon and polygonMumbai. I don't think this would matter much. wdyt? 🤷‍♂️

Reply:
> switch network modal, all the networks will show up.

It only shows the network that we're switching to

Reply:
we won't show switch network modal but trigger the switching programmatically using switchNetwork from wagmi hook. So I think it won't be an issue